### ISSUE TITLE
Usage with Docker: apiKey not set

### ISSUE THREAD

**Issue by Lurow** [Fekide/strapi-plugin-deepl#14]
Hi dev team,

thanks for this very nice plugin! 

I'm currently trying to build a stateless strapi production version on AWS Fargate using Docker.

So far everything works quite fine, but whenever I'm trying to include the DeepL Plugin, I get the following error:

` error: Error regarding deepl config: apiKey is not set, disable the plugin if you do not have one since the translations will not succeed`

I tried setting the environment variable on as a docker run parameter as well as in my AWS Fargate config. Somehow it's not consumed.

Any ideas?

 
Dockerfile:
```# BASE
ARG NODE_VERSION=14
FROM node:${NODE_VERSION}-alpine AS base-alpine
EXPOSE 1337

FROM base-alpine

ARG STRAPI_VERSION=latest

RUN yarn global add @strapi/strapi@${STRAPI_VERSION}


RUN mkdir -p /srv/app && chown 1000:1000 -R /srv/app

WORKDIR /srv/app

VOLUME /srv/app

COPY docker-entrypoint.sh /usr/local/bin/

RUN chmod 777 /usr/local/bin/docker-entrypoint.sh && ln -s /usr/local/bin/docker-entrypoint.sh

ENTRYPOINT ["docker-entrypoint.sh"]

CMD ["strapi"]
```

docker-entrypoint.sh:
```
#!/bin/sh
set -ea

if [ "$*" = "strapi" ]; then

  if [ ! -f "package.json" ]; then

    DATABASE_CLIENT=${DATABASE_CLIENT:-sqlite}

    EXTRA_ARGS=${EXTRA_ARGS}

    echo "Using strapi $(strapi version)"
    echo "No project found at /srv/app. Creating a new strapi project ..."

    DOCKER=true strapi new . --no-run \
      --dbclient=$DATABASE_CLIENT \
      --dbhost=$DATABASE_HOST \
      --dbport=$DATABASE_PORT \
      --dbname=$DATABASE_NAME \
      --dbusername=$DATABASE_USERNAME \
      --dbpassword=$DATABASE_PASSWORD \
      --dbssl=$DATABASE_SSL \
      $EXTRA_ARGS

  elif [ ! -d "node_modules" ] || [ ! "$(ls -qAL node_modules 2>/dev/null)" ]; then

    if [ -f "yarn.lock" ]; then

      echo "Node modules not installed. Installing using yarn ..."
      yarn install --prod --silent

    else

      echo "Node modules not installed. Installing using npm ..."
      npm install --only=prod --silent

    fi

  fi

  if [ "$NODE_ENV" = "production" ]; then
    STRAPI_MODE="start"
  elif [ "$NODE_ENV" = "development" ]; then
    STRAPI_MODE="develop"
  fi


  echo "Installing plugins..."
  yarn add @strapi/plugin-graphql @strapi/plugin-seo strapi-plugin-deepl

  echo "Rebuilding app to make env changes appear..."
  strapi build

  echo "Starting your app (with ${STRAPI_MODE:-develop})..."
  exec strapi "${STRAPI_MODE:-develop}"

else
  exec "$@"
fi
```

**Comment by sargreal** [comment]
Hi @Lurow :)

Thanks for your issue!

There was a major bug in parsing the environment variables, i have just published a fix for it in version v0.3.6!

Now you should be able to use the environment variable `DEEPL_API_KEY` as expected :)

**Comment by Lurow** [comment]
Wow that was fast *-*

Working like a charm :) Thank you so much!
